mealplanner-ui/src/pages/Meals/MealTags.tsx

```diff
@@ -0,0 +1,65 @@
+import { Chip, Grid, Stack } from "@mui/material";
+import { graphql } from "babel-plugin-relay/macro";
+import { useFragment } from "react-relay";
+import { MealTags_tags$key } from "./__generated__/MealTags_tags.graphql";
+import { setSelectedMealTags } from "../../state/state";
+
+export const MealTagsFragment = graphql`
+  fragment MealTags_tags on Query
+    @refetchable(queryName: "MealTagsRefetchQuery") {
+        gqLocalState {
+            selectedMealTags
+        }
+        allMealTags(first:100) {
+            edges {
+                node
+            }
+        }
+  }
+`;
+
+export const MealTags = ({ tags }: { tags: MealTags_tags$key }) => {
+  const mealTags = useFragment(MealTagsFragment, tags);
+  // If nothing is selected, mealTags.gqLocalState.selectedMealTags will return a null
```

Comment:
I can propose removing these comments since it doesn't add any value because the code is self-explanatory. People can update the code and they will forget to update the comment.

Reply:
I had implemented them in meal plan tags and explained it to Priya through comments in code. She has copy-pasted the same.